### ISSUE TITLE
Basic UI for authority search

### ISSUE THREAD

**Issue by fsteeg** [hbz/lobid-authorities#15]

**Comment by fsteeg** [comment]
Basic UI setup and search deployed to: http://test.lobid.org/authorities/search

Next steps:

- [ ] Useful short document representation in result list (types, names; depends on #3)
- [ ] Faceted search based on GND ontology (#8, depends on #2)

Will push current state to master, move this and #14 to backlog, move #2 and #3 to ready.

**Comment by fsteeg** [comment]
Deployed first item from https://github.com/hbz/lobid-authorities/issues/15#issuecomment-321581298 (useful short document representation in result list):
http://test.lobid.org/authorities/search?q=ostsee

**Comment by fsteeg** [comment]
Remaining part 2 of https://github.com/hbz/lobid-authorities/issues/15#issuecomment-321581298 actually has its own issue: #8. Closing this.
